documentation/API_documentation/Population-Density-Data-API-Readiness-Checklist.md

```diff
@@ -10,7 +10,7 @@ Checklist for population-density-data 0.1.1-rc.1 in release r1.1
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embed documentation into API spec - [link](/code/API_definitions/population-density-data.yaml) |
 |  6 | User stories                                 |   O   |         O         |    O    |    M   |  N   |      |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Pending   |  |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y  | Included in Test PR#44 |
```

Comment:
Need to be updated after the merge of #44 or you anticipate here the link.

Reply:
Yes, I didn't make any change in the table since #44 is also modifying that part of the file, avoiding conflicts.